src/constants/index.js

```diff
@@ -0,0 +1,75 @@
+import { Boat, Home, Human, Seeds, Shelter } from "@/icons";
+
+export const FLOOD_LEVEL = [
+  {
+    color: "#426CDB",
```

Comment:
Is it possible to set the color in `tailwind.config.mjs` and utilize it by name?